packages/salesforcedx-vscode-soql/package.json

```diff
@@ -61,19 +57,23 @@
     "@types/vscode": "1.49.0",
     "@typescript-eslint/eslint-plugin": "^3.7.1",
     "@typescript-eslint/parser": "^3.7.1",
+    "applicationinsights": "1.0.7",
     "chai": "^4.0.2",
     "cross-env": "5.2.0",
+    "debounce": "^1.2.0",
     "eslint": "6.0.1",
     "eslint-plugin-header": "^3.0.0",
     "eslint-plugin-jsdoc": "^30.0.3",
     "eslint-plugin-prettier": "^3.1.4",
     "mocha": "^5",
+    "papaparse": "^5.3.0",
     "sinon": "7.5.0",
-    "typescript": "3.8.3"
+    "typescript": "3.8.3",
+    "vscode-languageclient": "6.1.3"
   },
   "scripts": {
     "build": "npm run compile",
-    "compile": "tsc -p ./",
+    "compile": "tsc -p ./ && webpack",
```

Comment:
not sure if you need both tsc and webpack here since you're using the ts-loader, but could be wrong.

Reply:
We compile using tsc for the integration tests, which are not webpacked.

Reply:
Got it. Seems like both the tsc and webpacked source are being included when bundled with the vsix

Reply:
😱 Thanks. Just committed a fix for this.